Orange/statistics/util.py

```diff
@@ -476,6 +476,45 @@ def nanmean(x, axis=None, weights=None):
     return means
 
 
+def nan_mean_variance_axis(x, axis=None, weights=None):
```

Comment:
No, no particular reason, it just additional work and should also work differently. With `weights=None` the input weights should be a matrix. Otherwise I do not think the problem is clearly defined. On a related note, I do not think `nanmean` works properly with weights when axis is None.

Therefore I avoided it in this PR. What do you think?

Reply:
> with weights=None the input weights should be a matrix. [...] I do not think nanmean works properly with weights when axis is None.

I think you're right. It looks like the weights are just ignored. I would still prefer this function to be `nan_mean_variance`, so that if anyone, at some points, decides they need this computation with `axis=None`, we won't have to go through a renaming cycle, or end up with a second function with the practically same name. I'd be totally fine if this function raises a `NotImplementedError` if `axis is None`.

Reply:
Done in the last push.